data/hai/hai reveal 5 philosophers.txt

```diff
@@ -0,0 +1,1075 @@
+# Copyright (c) 2020 Nomadic Volcano
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+########################################################################
+
+# Part 2: Hai-Human Conflict
+
+mission "Hai Rescue [B01] Election"
+	landing
+	name "Missing Researchers"
+	description "Meet Teeneep on <destination>."
+	source "Darkwaste"
+	destination "Hai-home"
+	clearance
+	to offer
+		has "Hai Rescue [A09] Slaver Ships: done"
+	on offer
+		event "hai and human ships at wormhole"
+		conversation
+			`You land outside the hidden base and you notice the strangely-outfitted Centipede is gone. It has been replaced with a pair of Lightning Bugs which appear to have standard outfits. You walk into the base and see some researchers you saw before, but Teeneep is absent.`
+			`	You explain your discoveries and the Hai look increasingly concerned with each sentence you speak.`
+			`	After paying you, they explain, "Not long after you left, we went public with our story. We told everyone who would listen. Then the government received your findings through Earth authorities. There's been a big outpouring of support, and a lot of anger at the Elders for letting this go on for so long."`
+			`	"Two days ago, there was an election. A third of the Elders have been replaced, which is the fastest change we've had since the Korath invasion began. Teeneep is being inaugurated today, as the youngest 'elder' since the Elder Council was formed. I fear a time of chaos has begun."`
+			`	 "We'll contact her by hyperwave and tell her everything you told us. Please fly immediately to <destination>. The Elders will no doubt meet as soon as you arrive."`
```

Comment:
I kinda want to say it should be "We'll send her a hyperwave" or some such thing. There's probably an in-universe for the type of message, so it doesn't hurt to name it. I agree existing implementation was clunky though.

Reply:
```suggestion
			`You land outside the hidden base and notice the strangely-outfitted Centipede is gone. It has been replaced with a pair of Lightning Bugs which appear to have standard outfits. You walk into the base and see some researchers you saw before, but Teeneep is absent. You explain your discoveries, and the Hai look increasingly concerned with each sentence you speak.`
			`	After paying you, they explain, "Not long after you left, we went public with our story. We told everyone who would listen. Then the government received your findings through Earth authorities. There's been a big outpouring of support, and a lot of anger at the Elders for letting this go on for so long.`
			`	"Two days ago, there was an election. A third of the Elders have been replaced, which is the fastest change we've had since the Korath invasion began. Teeneep is being inaugurated today, as the youngest 'elder' since the Elder Council was formed. I fear a time of chaos has begun.`
			`	 "We'll send her a hyperwave and tell her everything you told us. Please fly immediately to <destination>. The Elders will no doubt meet as soon as you arrive."`
```
This one has hyperwave in it. I'll let you decide if you wanna just go with that or workshop it a bit more.